packages/admin/src/Charcoal/Admin/Script/Translation/TranslateScript.php

```diff
@@ -178,7 +178,7 @@ public function globRecursive($pattern, $flags = 0)
      */
     public function base()
     {
-        return realpath($this->app()->config()->get('base_path') . '../../../') . '/';
+        return realpath($this->app()->config()->get('base_path') . DIRECTORY_SEPARATOR . '../../../') . '/';
```

Comment:
Since this is a CLI script, the base path was probably defaulting to `/vendor/locomotivemtl/charcoal-app`.

Other locations in this file might be affected by the missing trailing slash.

```suggestion
        return realpath($this->app()->config()->get('base_path');
```